app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
+
+    if len(command) != 3:
+        return
+
+    if command[0] != "cp":
+        return
+    if command[1] == command[2]:
+        return
+
+    with (
+        open(command[1], "r") as file1,
+        open(command[2], "w") as file2
+    ):
+        file2.write(file1.read())
```

Comment:
You could use more specific variable naming, e.g. `file_copy`, `file_initial` instead of `file1` and `file2`